Translation corrections (#137)
07th-mod/higurashi-console-arcs: Update/some_012.txt

			NULL, "\"It's ok, Natsumi.", Line_WaitForInput);
	ModPlayVoiceLS(4, 0, "ps3/s04/00/310700071", 256, TRUE);
	OutputLine(NULL, "あとはお父さんに任せなさい。",
			NULL, " Leave the rest to Dad.", Line_WaitForInput);
	ModPlayVoiceLS(4, 0, "ps3/s04/00/310700072", 256, TRUE);
	OutputLine(NULL, "お前と春子は元通り、いつも通りに生活すればそれで、",

> 			NULL, " If you and Haruko can go back to your lives the way before, then-- ", Line_WaitForInput);

Reviewer: ```suggestion
			NULL, " If you and Haruko can go back to your lives the way before, then...", Line_WaitForInput);
```

	ModPlayVoiceLS(4, 0, "ps3/s04/00/310700073", 256, TRUE);
	OutputLine(NULL, "————ぐはッ？！」",

> 			NULL, " ...Guah?!\"", Line_Normal);

Reviewer: ```suggestion
			NULL, " Guah?!\"", Line_Normal);
```

	if (GetGlobalFlag(GADVMode)) {
		OutputLineAll("", NULL, Line_ContinueAfterTyping);
		OutputLine(NULL, "異様な気配に、恐る恐る見上げる。…その先にはおかーさんが血のしたたる包丁を手にして、…さっきまで動いて、しゃべって、息をしていたおとーさんを、…傲然と見下ろしていた…。",

> 				NULL, "<size=-2>I gawked up in fear at the bizarre presence behind Dad. Mom was right there, holding a kitchen knife dripping with blood... She stood arrogantly, looking down at Dad... who had been moving, talking and breathing just a moment ago...", Line_Normal);

Reviewer: ```suggestion
				NULL, "<size=-2>I gawked in horror at the bizarre presence behind Dad. Mom was right there, holding a kitchen knife dripping with blood... She stood arrogantly, looking down at Dad... who had been moving, talking and breathing just a moment ago...", Line_Normal);
```

	} else {
		OutputLine(NULL, "異様な気配に、恐る恐る見上げる。…その先にはおかーさんが血のしたたる包丁を手にして、…さっきまで動いて、しゃべって、息をしていたおとーさんを、…傲然と見下ろしていた…。",

> 				NULL, "I gawked up in fear at the bizarre presence behind Dad. Mom was right there, holding a kitchen knife dripping with blood... She stood arrogantly, looking down at Dad... who had been moving, talking and breathing just a moment ago...", Line_Normal);

Reviewer: ```suggestion
				NULL, "I gawked in horror at the bizarre presence behind Dad. Mom was right there, holding a kitchen knife dripping with blood... She stood arrogantly, looking down at Dad... who had been moving, talking and breathing just a moment ago...", Line_Normal);
```

	if (GetGlobalFlag(GADVMode)) { OutputLine("<color=#f5e6d3>夏美の母</color>", NULL, "<color=#f5e6d3>Natsumi's mom</color>", NULL, Line_ContinueAfterTyping); }
	ModPlayVoiceLS(4, 0, "ps3/s04/00/310600181", 256, TRUE);
	OutputLine(NULL, "「みんなが悪いの。",
			NULL, "\"It's everyone else's fault.", Line_WaitForInput);
	ModPlayVoiceLS(4, 0, "ps3/s04/00/310600182", 256, TRUE);
	OutputLine(NULL, "みんなが協力してくれないから悪いの」",
			NULL, " It's everyone else's fault for not cooperating.\"", Line_Normal);
	ClearMessage();

//夏美の母rvS04/00/310600183.「オヤシロさまの祟りだと騒いだり、死体をわざわざ見つかるようなところに捨てたりしてkvS04/00/310600184.………みんなが、家族が、私の邪魔をするのッ！kvS04/00/310600185.　みんな、私の家族が悪いのよッ！」
	if (GetGlobalFlag(GADVMode)) { OutputLine("<color=#f5e6d3>夏美の母</color>", NULL, "<color=#f5e6d3>Natsumi's mom</color>", NULL, Line_ContinueAfterTyping); }
	ModPlayVoiceLS(4, 0, "ps3/s04/00/310600183", 256, TRUE);
	OutputLine(NULL, "「オヤシロさまの祟りだと騒いだり、死体をわざわざ見つかるようなところに捨てたりして",

> 			NULL, "\"Making a racket about Oyashiro-sama's curse, dumping  corpses in easy to find locations.", Line_WaitForInput);

Reviewer: ```suggestion
			NULL, "\"Making a racket about Oyashiro-sama's curse, dumping corpses in easy to find locations.", Line_WaitForInput);
```
